velox/exec/tests/utils/PlanBuilder.cpp

```diff
@@ -892,6 +893,58 @@ PlanBuilder& PlanBuilder::localMerge(const std::vector<std::string>& keys) {
   return *this;
 }
 
+PlanBuilder& PlanBuilder::expand(
+    const std::vector<std::vector<std::string>>& projections) {
+  std::vector<std::vector<core::TypedExprPtr>> projectExprs;
+  VELOX_CHECK(!projections.empty(), "projections must be not null.");
+  projectExprs.reserve(projections.size());
+  const auto numColumns = projections[0].size();
+  const auto numRows = projections.size();
+  std::vector<std::string> names;
+  names.reserve(numColumns);
+  std::vector<TypePtr> types;
+  types.reserve(numColumns);
+  const auto& inputRowType = planNode_->outputType();
+  auto groupIdColumnType = BIGINT();
```

Comment:
We have discussed that Expand node should not hard-code 'group-id' logic. Please, revise.